### PR TITLE
update yarn to 0.18.1

### PR DESCRIPTION
**Pull request by aliceriot**
Fix for a deploy problem on CI, relevant yarn issues:

https://github.com/yarnpkg/yarn/issues/2142
https://github.com/heroku/heroku-buildpack-nodejs/issues/359
https://github.com/yarnpkg/yarn/issues/1981

### CODE REVIEW

File: travis/Dockerfile-travis-watch [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM mitodl/mm_watch_travis_yarn
+FROM mitodl/mm_watch_travis_yarn_update
```
Comment (noisecapella) [inline travis/Dockerfile-travis-watch]: What's the change here?

Comment (aliceriot) [inline travis/Dockerfile-travis-watch]: I'm going to undo that - I was thinking we might want to point things at a different container, but I don't think we need to.

**Comment by noisecapella** [comment]
:+1: 

Can you update the description to say that this is to fix a deploy problem on CI?

**Comment by aliceriot** [comment]
yea

**Pull request merged**